packages/api-client/src/tcp/WebSocketClient.test.ts

```diff
@@ -58,83 +58,85 @@ const fakeSocket = {
 describe('WebSocketClient', () => {
   describe('handler', () => {
     it('calls "onOpen" when WebSocket opens', async () => {
-      const websocketClient = new WebSocketClient('url', fakeHttpClient);
-      const onOpenSpy = spyOn<any>(websocketClient, 'onOpen').and.callThrough();
+      const websocketClient = new WebSocketClient('ws://url', fakeHttpClient);
+      const onOpenSpy = jest.spyOn(websocketClient as any, 'onOpen');
```

Comment:
maybe destructuring makes sense in this file also since its called many times. also wondering about all these `as any` blocks if they can be improved

Reply:
they sadly cannot be improved easily since the test is relying on mocking private methods :/ 
This would be too much of a hustle trying to fix this, so I'm restoring to the original behavior